README.md

```diff
@@ -164,6 +163,31 @@ plugins: [
 ]
 ```
 
+## Events
+
+To allow other plugins catch and customise the result of Favicons Library before this plugin emits all the assets, this plugin executes the following events:
+
+### AsyncSeriesWaterfallHook
+
+* `webappWebpackPluginBeforeEmit`
```

Comment:
I think it would be better to make the hook name the title of the section and specify its kind below, something like this:

 ```md

### webappWebpackPluginBeforeEmit

> AsyncSeriesWaterfallHook
```